routing/router.go

```diff
@@ -1838,6 +1840,8 @@ func (r *ChannelRouter) sendPayment(payment *LightningPayment,
 			// newly updated fees.
 			case *lnwire.FailFeeInsufficient:
```

Comment:
This is not the only case where we can get back a channel update. For example, `FailIncorrectCltvExpiry`. I think we should try to update `additionalEdges` also in those cases.

Reply:
Can the update maybe be done inside `applyChannelUpdate` ?

Reply:
Done.